### PR TITLE
[WIP] Add samples

### PR DESCRIPTION
**Pull request by richlander**
Samples exist in https://github.com/microsoft/dotnet-framework-docker-samples/ today. We're going to close that repo, just like we did with https://github.com/dotnet/dotnet-docker-samples/.

- [x] .NET Framework console sample (dotnetapp)
- [x] ASP.NET Web Forms sample (aspnetapp)
- [x] ASP.NET MVC sample (aspnetmvcapp)
- [x] Samples README
- [x] Samples README for DockerHub

### CODE REVIEW

File: samples/README.DockerHub.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Latest Version of Common Tags
+
+The following tags are the latest stable versions of the most commonly used images. The complete set of tags is listed further down.
+
+- [`4.7.1`](https://github.com/Microsoft/dotnet-framework-docker/blob/master/4.7.1-windowsservercore-1709/runtime/Dockerfile)
```
Comment (MichaelSimons) [inline samples/README.DockerHub.md]: I assume we will want to push both the dotnetapp as well as the aspnetapp.  Should the tags be `dotnetapp-3.5` and `dotnetapp-4.7.2` and mirrored for aspnetapp?

File: samples/README.DockerHub.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+The [.NET Framework Docker samples](https://github.com/Microsoft/dotnet-framework-docker/blob/master/samples/README.md) show various ways to use .NET Framework and Docker together. See [Building Docker Images for .NET Core Applications](https://docs.microsoft.com/dotnet/framework/docker/) to learn more.
+
+Watch [dotnet/announcements](https://github.com/dotnet/announcements/labels/Docker) for Docker-related .NET announcements.
+
+### Container sample: Run a simple application
+
+Type the following command to run a sample console application:
+
+```console
+docker run --rm microsoft/dotnet-framework-samples
```
Comment (MichaelSimons) [inline samples/README.DockerHub.md]: I just noticed we don't document what `latest` refers to here as well as the .NET Core samples.

File: samples/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+# .NET Framework Docker Samples
+
+The samples show various ways to use .NET Framework and Docker together. You can use the samples as the basis of your own Docker images or just to play.
+
+The samples exercise various levels of functionality. The [.NET Framework Docker sample](dotnetapp/README.md) includes the most functionality, including build, unit testing, and pushing images to a container registry. The [ASP.NET Core Docker sample](aspnetapp/README.md) includes instructions for testing images with [Azure Container Instances](https://azure.microsoft.com/services/container-instances/). The samples include detailed instructions for use with and without Docker.
```
Comment (MichaelSimons) [inline samples/README.md]: I see a lot of `Core` refernces throughout the docs.

File: samples/dotnetapp/dotnet-docker-dev-in-container.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+ 2. Add a [Directory.Build.props](Directory.Build.props) file to your project to use a different set of `obj` and `bin` folders for your local and container environments.
+
+## Run your application in a container while you Develop
+
+You can rerun your application in a container with every local code change. This scenario works for both console applications and websites.
+
+The instructions assume that you are in the root of the repository.
+
+```console
+docker run --rm -it -v c:\git\dotnet-docker\samples\dotnetapp:c:\app\ -w \app\dotnetapp microsoft/dotnet-framework:4.7.2-sdk dotnet watch run
```
Comment (MichaelSimons) [inline samples/dotnetapp/dotnet-docker-dev-in-container.md]: `dotnet-docker` reference should be `dotnet-framework-docker`

File: samples/README.md (hunk 3 above)
Comment (richlander) [reply samples/README.md]: Yes. Those are the docs I started with. I think I got most of the "Core" references.

**Pull request closed without merging**